Todo.py

```diff
@@ -0,0 +1,203 @@
+import discord
+from settings import *
+from checks import *
+
+TOKEN = "NTY4MDkyNTY5NTE5NzgzOTM2.XMiBDw.zCL0KhV_gOtxfyACctgYCLcLlEY"
+# PREFIX = "g:"
+
+# EMOJIS = {
+#     "wastebasket": "\U0001F5D1",
+#     "check": "\U00002705",
+#     "hourglass": "\U000023F3"
+# }
+
+#bot = commands.Bot(command_prefix=PREFIX)
+
+class Todo(commands.Cog):
+    """A suite of command to make a nice todo list."""
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        if message.author == bot.user:
+            return
+        await bot.process_commands(message)
+
+    @commands.Cog.listener()
+    async def on_raw_reaction_add(self, reaction):
+        if reaction.user_id != bot.user.id:
+            message = await bot.get_channel(reaction.channel_id).fetch_message(reaction.message_id)
+
+            with open(TODO_CHANNEL, "r") as file:
+                channelId = file.readlines()[0]
+
+            if reaction.channel_id == int(channelId): #Check if it's a todo-message (check if it's the good channel)
+                if len(message.embeds) > 0: # Check if it's an embed, I think this will avoid most problems
+                    if reaction.emoji.name == EMOJIS['wastebasket']:
+                        await bot.get_channel(reaction.channel_id).delete_messages([message])
+
+                        repostFieldValue = None
+                        for field in message.embeds[0].fields:
+                            if field.name == "Public repost":
```

Comment:
instead of having a single allowed name "Public repost" have global var so that is can be configured once I implement #4

Reply:
What's the goal ?

Reply:
one day the bot might extend its reach to several servers so I want to keep it ready for such an eventuality.

Reply:
As we now know that the bot is going to be used in more than one server, I'll  be waiting for you to fix this.

Reply:
I made a PUBLIC_REPOST constant, even if I'm not sure if it was what you wanetd

Reply:
:man_facepalming: Sry I'm such an idiot. I read through your code a little too fast it would seem. I completely missed misunderstood the use of this thing. It probably wasn't needed. Sry I made a big mistake here :grimacing:

Reply:
Oh, so I remove it ? We can also let it in memory to make translation xD

Reply:
Do whatever you want I don't have my word in this matter anymore...